core/controllers/admin.py

```diff
@@ -1554,6 +1555,34 @@ def put(self) -> None:
         user_services.set_username(user_id, new_username)
         user_services.log_username_change(
             self.user_id, old_username, new_username)
+
+        # Update profile picture.
+        old_fs = fs_services.GcsFileSystem(
+            feconf.ENTITY_TYPE_USER, old_username)
+        new_fs = fs_services.GcsFileSystem(
+            feconf.ENTITY_TYPE_USER, new_username)
+        if old_fs.isfile('profile_picture.png'):
+            image_png = old_fs.get('profile_picture.png')
+            old_fs.delete('profile_picture.png')
+            new_fs.commit(
+                'profile_picture.png', image_png, mimetype='image/png')
+        else:
+            raise Exception(
+                'The user with username %s does not have a '
+                'profile picture with png extension.' % (old_username)
+            )
```

Comment:
Hmm, should this be just a warning?

Reply:
What else should we do? Do we have to return something else?